src/core/border_router/routing_manager.hpp

```diff
@@ -198,6 +198,8 @@ class RoutingManager : public InstanceLocator
     static constexpr uint32_t kRtrSolicitationInterval     = 4;   // Interval between RSs. In sec.
     static constexpr uint32_t kMaxRtrSolicitationDelay     = 1;   // Max delay for initial solicitation. In sec.
     static constexpr uint32_t kMaxRoutingPolicyDelay       = 1;   // Max delay for routing policy evaluation. In sec.
+    static constexpr uint32_t kRtrSolicitationRetryFailDelay =
+        60; // The delay before re-trying sending failed Router Solicitations. In Sec.
```

Comment:
How about renaming the constant name and making the comment shorter so it fits in same line (so it looks nicer)?
Something like this maybe:
```suggestion
    static constexpr uint32_t kRtrSolicitationRetryDelay = 60; // Delay to re-try  failed Router Sol. tx. In Sec.
```

Reply:
ha, it's not pretty :(
Updated!